client/quantum_serverless/core/provider.py

```diff
@@ -455,19 +455,58 @@ class IBMServerlessProvider(ServerlessProvider):
     """
     A provider for connecting to the IBM serverless host.
 
-    Example:
-        >>> provider = IBMServerlessProvider(token="<TOKEN>")
+    Credentials can be saved to disk by calling the `save_account()` method::
+
+        from quantum_serverless import IBMServerlessProvider
+        IBMServerlessProvider.save_account(token=<INSERT_IBM_QUANTUM_TOKEN>)
+
+    Once the credentials are saved, you can simply instantiate the provider with no
+    constructor args, as shown below.
+
+        from quantum_serverless import IBMServerlessProvider
+        provider = IBMServerlessProvider()
+
+    Instead of saving credentials to disk, you can also set the environment variable
+    ENV_GATEWAY_PROVIDER_TOKEN and then instantiate the provider as below::
+
+        from quantum_serverless import IBMServerlessProvider
+        provider = IBMServerlessProvider()
+
+    You can also enable an account just for the current session by instantiating the
+    provider with the API token::
+
+        from quantum_serverless import IBMServerlessProvider
+        provider = IBMServerlessProvider(token=<INSERT_IBM_QUANTUM_TOKEN>)
     """
 
-    def __init__(self, token: str):
+    def __init__(self, token: Optional[str] = None, name: Optional[str] = None):
         """
         Initialize a provider with access to an IBMQ-provided remote cluster.
 
         Args:
             token: IBM quantum token
+            name: Name of the account to load
         """
+        token = token or IBMProvider(name=name).active_account().get("token")
         super().__init__(token=token, host=IBM_SERVERLESS_HOST_URL)
 
+    @classmethod
+    def save_account(
+        cls,
+        token: Optional[str] = None,
+        name: Optional[str] = None,
+        overwrite: Optional[bool] = False,
+    ) -> None:
+        """
+        Save the account to disk for future use.
+
+        Args:
+            token: IBM Quantum API token
+            name: Name of the account to save
+            overwrite: ``True`` if the existing account is to be overwritten
+        """
+        IBMProvider.save_account(token=token, name=name, overwrite=overwrite)
```

Comment:
Here we have a subset of input arguments used in `IBMProvider.save_account`. The other arguments did not seem relevant to our software, so I left them out. We will just pass the inputs from this function "blindly" into `IBMProvider.save_account` and let `IBMProvider` handle the error messaging.